pipeline/src/lib.rs

```diff
@@ -169,22 +175,6 @@ pub fn handle_simple_queries_callback<'a, T: FieldElement>() -> impl QueryCallba
         let (id, data) = parse_query(query)?;
         match id {
             "None" => Ok(None),
-            "Output" => {
```

Comment:
@pacheco here

Reply:
oh right, my bad, i must have looked into the old version